CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to Tombolo Digital Connector
+
+Thank you for choosing to contribute to this project. **We need your superpowers!**  
+To learn more about the Tombolo vision checkout our [website](http://www.tombolo.org.uk/).
```

Comment:
Potentially change checkout to check out and leave an empty line above it for aesthetic reasons.